c++/src/Reader.hh

```diff
@@ -142,6 +143,30 @@ namespace orc {
 
     // row index of current stripe with column id as the key
     std::unordered_map<uint64_t, proto::RowIndex> rowIndexes;
+    std::map<uint32_t, BloomFilterIndex> bloomFilterIndex;
```

Comment:
Shall we remove this in this PR because this seems to be unused in this PR?

Reply:
it is used to hold bloom filter index in Reader.cc (e.g. line 378)